src/icons/folderIcons.ts

```diff
@@ -82,14 +82,14 @@ export const folderIcons: FolderTheme[] = [
             { name: 'folder-include', folderNames: ['include', 'includes', '_includes'] },
             { name: 'folder-docker', folderNames: ['docker', 'dockerfiles', '.docker'] },
             { name: 'folder-ngrx-effects', folderNames: ['effects'], enabledFor: [IconPack.Ngrx] },
-            { name: 'folder-ngrx-store', folderNames: ['store'], enabledFor: [IconPack.Ngrx] },
+            { name: 'folder-ngrx-store', folderNames: ['stores', 'store'], enabledFor: [IconPack.Ngrx] },
             { name: 'folder-ngrx-state', folderNames: ['states', 'state'], enabledFor: [IconPack.Ngrx] },
             { name: 'folder-ngrx-reducer', folderNames: ['reducers', 'reducer'], enabledFor: [IconPack.Ngrx] },
             { name: 'folder-ngrx-actions', folderNames: ['actions'], enabledFor: [IconPack.Ngrx] },
             { name: 'folder-ngrx-entities', folderNames: ['entities'], enabledFor: [IconPack.Ngrx] },
             { name: 'folder-redux-reducer', folderNames: ['reducers', 'reducer'], enabledFor: [IconPack.Redux] },
             { name: 'folder-redux-actions', folderNames: ['actions'], enabledFor: [IconPack.Redux] },
-            { name: 'folder-redux-store', folderNames: ['store'], enabledFor: [IconPack.Redux] },
+            { name: 'folder-redux-store', folderNames: ['stores', 'store'], enabledFor: [IconPack.Redux] },
```

Comment:
Hi @imbsky, I do not really want to merge this. If you have a "stores" folder in a Redux based project you might be doing something wrong, because the concept of Redux basically suggests you to use only a single store (so "store"). I've read that you are using MobX where multiple stores are possible, but here we are talking about the Redux IconPack (same for NgRx). Feel free to configure this locally in your user settings like this:
```json
"material-icon-theme.folders.associations": {
    "stores": "redux-store"
}
```

Hopefully, that's okay for you 😄

Reply:
Oh, that solution seems good. and I don't write TypeScript anymore so I have no objection.